eZ/Publish/Core/REST/Common/FieldTypeProcessor/RelationListProcessor.php

```diff
@@ -39,4 +39,17 @@ function ($contentId) {
 
         return $outgoingValueHash;
     }
+
+    public function postProcessFieldSettingsHash($outgoingSettingsHash)
+    {
+        $outgoingSettingsHash = parent::postProcessFieldSettingsHash($outgoingSettingsHash);
+
+        if (isset($outgoingSettingsHash['selectionDefaultLocation'])) {
+            $outgoingSettingsHash['selectionDefaultLocationHref'] = $this->mapToContentHref(
```

Comment:
You can't use `mapToContentHref` here: the id we are mapping is a location id, not a content id. The url needs to be generated using the `ezpublish_rest_loadLocation` route, with a `locationId` parameter.